etl/hhc_hospitals.py

```diff
@@ -0,0 +1,42 @@
+from dataflows import *
+from lib import dump_to_postgis, rename_field, map_field
+import os
+import csv
+import sys
+from pathlib import Path
+from utils import url, geo_flow, get_the_geom, quick_clean, get_hnum, get_sname
+
+csv.field_size_limit(sys.maxsize)
+
+table_name = 'hhc_hospitals'
+
+hhc_hospitals = Flow(
+    load(url, resources = table_name, force_strings=False),
+    add_field('datasource', 'string', table_name),
+    map_field('location_1', operation=lambda a: quick_clean(a)),
```

Comment:
you can make the code more standardized by doing :
```
add_computed_field([dict(target=dict(name = 'address', type = 'string'),
                                operation=lambda row: quick_clean(row['location_1'])
                               ])
```
This can be appended with the following add_computed_field().